tests/schemas/invalid/test_invalid_q_code_regex_pattern.json

```
@@ -0,0 +1,63 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.1",
+  "survey_id": "0",
+  "title": "q_codes on different blocks",
+  "theme": "default",
+  "description": "A questionnaire to demo q_codes in answers for data version 0.0.1",
```

The only thing left is correct description of this schema
```suggestion
  "description": ""A questionnaire to test q_code regex pattern for an answer"",
```
?